src/main/java/com/fatec/springAPI4/entity/Expertise.java

```diff
@@ -0,0 +1,31 @@
+package main.java.com.fatec.springAPI4.entity;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.Setter;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+
+@Getter @Setter @AllArgsConstructor
+@Entity
+@Table
+public class Expertise {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = ex_id)
+    private Long id;
+
+    @Column(name = ex_name)
```

Comment:
`ex_id` e `ex_name` nessas anotações `@Column` não precisam das aspas?

```suggestion
    @Column(name = "ex_id")
    private Long id;

    @Column(name = "ex_name")
```

Reply:
corrigido